doc/src/site/sphinx/migration_guide.rst

```diff
@@ -229,9 +229,12 @@ Removal of Deprecated Methods and Classes
 
 - In ``H2OAutoML`` Python and Scala API, the member ``leaderboard()``/``leaderboard`` is replaced by the method ``getLeaderboard()``.
 
-- The method ``setClusterConfigFile`` was removed from ``H2OConf`` in Python and Scala API. The replacement method is
+- The method ``setClusterConfigFile`` was removed from ``H2OConf`` in Scala API. The replacement method is
   ``setClusterInfoFile`` on ``H2OConf``.
 
+- The method ``setClientPortBase`` was removed from ``H2OConf`` in  Scala API. The replacement method is
```

Comment:
I don't whether a comment about `setClientPortBase` was removed on purpose or by resolving some conflicts during merge. Just adding it for completeness.

Reply:
I haven't removed it on purpose, If I did, it was merge accident :(.

Thank you for fixing!